### PR TITLE
get product comparisons for all root files; clean up debugs in logRootQA.py

### PR DESCRIPTION
**Pull request by slava77**
all updates are in logRootQA.py
- change the file selection to provide a list of changes in products for all available files
- also, cleanup debug printouts in functions

### CODE REVIEW

File: logRootQA.py [hunk 1]
```diff
@@ -319,7 +319,8 @@ def summaryComp(compDir):
 for r in commonRoots:
 #    print 'I could have tested',r
     if ('PU' in r or 'RECODR' in r or 'REMINIAOD' in r) and 'DQM' not in r:
-        sameEvts=sameEvts and checkEventContent(baseDir+r,testDir+r)
```
Comment (slava77) [inline logRootQA.py:322]: @davidlange6 
(as original developer)
was it intentional to have a check only until the first file difference?
As written, after `sameEvts` is False, the following statements are not evaluated.

**Comment by slava77** [comment]
@cmsbuild please test with cms-sw/cmssw#31220

**Comment by slava77** [comment]
@cmsbuild please abort test

I'll pick something easier to test, it looks like this one does not pick up dependencies

**Comment by slava77** [comment]
@cmsbuild please test with cms-sw/cmssw#31985

**Comment by slava77** [comment]
@cmsbuild please abort test

**Comment by slava77** [comment]
@cmsbuild please test with cms-sw/cmssw#31220,cms-data/RecoEgamma-ElectronIdentification#15

**Comment by slava77** [comment]
@cmsbuild please test with cms-sw/cmssw#31220,cms-data/RecoEgamma-ElectronIdentification#15,cms-sw/cmssw#32307

**Comment by smuzaffar** [comment]
@slava77 , please do not start the tests yet with cms-sw/cmssw#32307 , as there is already a test running for that PR, so start it here will abort the existing job. I would suggest to waot for cms-sw/cmssw#32307 and https://github.com/cms-sw/cmssw/pull/32313 to get merged first

**Comment by slava77** [comment]
@cmsbuild please test with cms-sw/cmssw#31220

**Comment by slava77** [comment]
I was naive to assume that merging https://github.com/cms-data/RecoEgamma-ElectronIdentification/pull/15 meant that it enough to make it to the IB, apparently https://github.com/cms-sw/cmsdist/pull/6456 still remains open

**Comment by slava77** [comment]
@cmsbuild please test with cms-sw/cmssw#31220,cms-data/RecoEgamma-ElectronIdentification#15

**Comment by slava77** [comment]
@cmsbuild please test with cms-sw/cmssw#31220

**Comment by slava77** [comment]
> @slava77 , are you happy with this change and its results?

https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_11_3_X_2020-11-30-1100+9191bb/40207/validateJR/logRootQA.log

IMHO it's a significant improvement: 
- the debug(-looking) file lists are out and the file size reduced from 427K to 223K
- while the more detailed information about the file products comparisons is now present

I'm happy with the results for now.

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**